src/appsignal/client.py

```diff
@@ -16,5 +16,6 @@ def __init__(self, **options: Unpack[Options]):
         self._options = DEFAULT_CONFIG | from_system() | options | from_public_environ()
 
     def start(self):
-        start_agent(self._options)
-        start_opentelemetry(self._options)
+        if self._options["active"]:
```

Comment:
Note that, because there is no default value for the `active` key, it can be `None`, and `self._options["active"]` will throw a `KeyError`. Use `self._options.get("active")` instead, or provide a default value for `active`.

Reply:
Updated :)